static/css/styles.css

```diff
@@ -178,67 +182,31 @@ footer {
     color: #3e001fa8;
 }
 
-/* menu modal images */
+/* Menu Modal Images */
 .modal img {
     max-height: 400px;
     width: auto;
     display: block;
     margin: 0 auto;
 }
 
-/* Forms css sign-in sign-out forgot password register */
-
-.form-signup-heading {
-    margin-bottom: 1rem;
-}
-
-.form-signup {
+/* Forms */
+.form-container {
     max-width: 600px;
     background: #fff;
     padding: 2rem;
     border-radius: 0.5rem;
     border: 1px solid #ced4da;
     box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
-    margin: auto;
-}
-
-.form-account-login {

```

Comment:
looks like we're removing a lot of CSS for the forms (register/login)... how come?

Reply:
Again, not intentional - what to do to avoid screwing up good work of others please?